extensions/scarb-doc/src/types.rs

```diff
@@ -21,6 +23,21 @@ use cairo_lang_syntax::node::{ast, SyntaxNode};
 
 use crate::db::ScarbDocDatabase;
 
+type ModuleUsesItems = (
+    Vec<ConstantId>,
+    Vec<FreeFunctionId>,
+    Vec<StructId>,
+    Vec<EnumId>,
+    Vec<ModuleTypeAliasId>,
+    Vec<ImplAliasId>,
+    Vec<TraitId>,
+    Vec<ImplDefId>,
+    Vec<ExternTypeId>,
+    Vec<ExternFunctionId>,
+    Vec<SubmoduleId>,
+    Vec<CrateId>,
+);
```

Comment:
I would enclose this and the extraction logic into a new struct instead, for better readability.  
For instance something like this: https://pastebin.swmansion.com/?f93d580e1791f3fc#2QyVDKnmrJyPnwdQjY3kQhnbgecmaFcksy9LibEoD13v